examples/nlp/intent_slot_classification/data/prompt_learning_assistant_preprocessing.py

```diff
@@ -0,0 +1,184 @@
+import argparse
```

Comment:
I put the preprocessing scripts each in the example directories for their task because I noticed that's where the intent and slot notebook puts their preprocessing scripts. The `prompt_learning_assitant_preprocessing` depends on the `assistant_utils.py` script that was already in the intent and slot example dir, so I think I would need to move them both if I move them to the `scripts/` dir

Reply:
Yeah. I think `scripts/` is the place that these data processing scripts belong to since they are not really the code for NeMo NLP

Reply:
Okay, I'll move them and update the Intent and Slot notebook so it doesn't break